closure/bin/build/jscompiler.py

```diff
@@ -46,7 +46,7 @@ def _ParseJavaVersion(version_string):
   """
   match = _VERSION_REGEX.search(version_string)
   if match:
-    version = tuple(int(x, 10) for x in match.groups())
+    version = tuple(int(x or '0', 10) for x in match.groups())
```

Comment:
I'd assume that  @shicks meant that he'd prefer: `int(x or 0)` which will not throw a TypeError since no base was given. I did like your previous solution too, except for the unnecessary negation.